pkg/apis/fleet.cattle.io/v1alpha1/bundle.go

```diff
@@ -31,6 +31,8 @@ var (
 	}
 )
 
+const DefaultWindow = "1h"
```

Comment:
We should put this in the Helm chart.

Reply:
@nickgerace do you want to expose this default Window as an option for the agent controller? similar to checkin interval?

Reply:
Yes!